resources/views/components/post-tile.blade.php

```diff
@@ -0,0 +1,13 @@
+<article class="{{ $class }} post-tile">
+  <header class="post-tile__header">
+    <h4 class="post-tile__heading">
+      <a href="{{ $link }}" class="post-tile__link">
+        {{ $title }}
+      </a>
+    </h4>
+  </header>
+
+  <p class="post-tile__excerpt">
```

Comment:
Sometimes excerpts might not exist so we will keep empty paragraphs in such cases. Shouldn't we show excerpt only when exists?